Lib/_pyio.py

```diff
@@ -2523,9 +2523,14 @@ def read(self, size=None):
                 size = size_index()
         decoder = self._decoder or self._get_decoder()
         if size < 0:
+
+            b = self.buffer.read()
```

Comment:
personal nitpick / nit: I like longer than one character variable names

Reply:
I usually prefer longer variable names as well! I was using `b` as a shorthand for `bytes` to avoid shadowing the global variable. 😅 Since I was quite tired at the time, I didn't fully consider the naming. With a fresh perspective, I’ve now chosen a more suitable name. I hope this works better!

Reply:
I usually prefer longer variable names as well! I was using `b` as a shorthand for `bytes` to avoid shadowing the global variable. 😅 Since I was quite tired at the time, I didn't fully consider the naming. With a fresh perspective, I’ve now chosen a more suitable name. I hope this works better!